src/adapters/repositories/userModel.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+export interface IUser extends Document {
```

Comment:
This information should be in a "User" entity in domain and you should create a IUser extending from it.